src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -216,8 +216,8 @@ def get_attrs(self, pathname):
             attrs["normalization"] = None
 
         try:
-            case_insensitive = self.fs.getxattr(pathname, 'ceph.dir.caseinsensitive').decode('utf-8')
-            attrs["case_insensitive"] = case_insensitive == "0"
+            casesensitive = self.fs.getxattr(pathname, 'ceph.dir.casesensitive').decode('utf-8')
+            attrs["case_insensitive"] = casesensitive == "0"
```

Comment:
These are typos? Seems like the logic or the means of storage changed at some point?

Reply:
The typo is in the xattr name. I fixed the variable name for consistency and easy grepping.